alf/networks/encoding_networks.py

```diff
@@ -248,7 +249,7 @@ def forward(self, inputs):
 
 
 @gin.configurable
-class EncodingNetwork(nn.Module):
+class MLPNetwork(nn.Module):
```

Comment:
Do you think it's necessary to have an MLP that just handles unnested input?

Reply:
`EncodingNetwork` can handle this casebut when we need preprocessing_layers to pre process multi inputs , we pass in (nested `EncodingNetwork`) without `preprocessing_layers` and `preprocessing_combiner` as `preprocessing_layers`